userspace/libsinsp/test/filtercheck_has_args.ut.cpp

```diff
@@ -68,7 +68,7 @@ TEST(filtercheck_has_args, has_args)
 				}
 				else if (field_str == "k8s.pod.label")
 				{
-					field_str += ".foo";
+					field_str += "[foo]";
 				}
```

Comment:
I think we can directly remove this since we can fall in the default case below, now this field doesn't require an explicit `.*` syntax